lib/node_modules/@stdlib/math/base/special/hyp2f1/docs/types/test.ts

```diff
@@ -0,0 +1,76 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import hyp2f1 = require( './index' );
+
+
+// TESTS //
+
+// The function returns a number...
+{
+	hyp2f1( 1.0, 1.0, 1.0, 0.0 ); // $ExpectType number
+	hyp2f1( 10.0, 7.4, -1.8, -0.99 ); // $ExpectType number
+	hyp2f1( 10.0, 1.0, -1.8, -0.99 ); // $ExpectType number
+	hyp2f1( 2.0, 3.0, 5.0, 0.99 ); // $ExpectType number
+}
+
+// The compiler throws an error if the function is provided a first argument other than a number...
```

Comment:
Rather than "other than a number" can we replace with "which is a not a number" here and below?